### ISSUE TITLE
ALTER TABLE and CREATE INDEX statements do not add schema to table name.

### ISSUE THREAD

**Issue by erwinkn** [drizzle-team/drizzle-kit-mirror#40]
**Reproduction:**

1. Define the following schema:
```ts
import { integer, pgSchema, serial, text, uniqueIndex } from "drizzle-orm/pg-core";

export const mySchema = pgSchema("my_schema");

export const table1 = mySchema.table(
  "table1",
  {
    id: serial("id").primaryKey(),
    text: text("text").notNull(),
  },
  (table1) => {
    return {
      uniqueIdx: uniqueIndex().on(table1.text),
    };
  }
);

export const table2 = mySchema.table("table2", {
  id: serial("id").primaryKey(),
  fk: integer("fk").references(() => table1.id),
});
```

2. Run `drizzle-kit generate:pg`
3. The `ALTER TABLE` statement for the foreign key and the `CREATE UNIQUE INDEX` statement reference `table1` and `table2` without appending the schema name.

**Expected behavior**
All table references in the generated migration include the schema name.

**Package versions**
`drizzle-orm`: 0.25.3
`drizzle-kit`: 0.17.6

**Comment by AdditionAddict** [comment]
Commenting to track, facing same issue

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.18.1`
